### PR TITLE
minimum donation notification

### LINKED ISSUE #694: Show minimum donation amount
**Describe the feature you'd like to see included in Plant-for-the-Planet Web App.**
Show the minimum amount for a donation we can process:
Screen Shot 2020-12-20 at 1 56 54 AM

**What are the use cases for this feature?**
The API does not allow a donation with too little amount. To prevent the users from trying to donate amounts not supported and getting an error message, we should inform them ahead.

### PR DESCRIPTION
**Pull request by captainamiedi**
Fixes #694 

Changes in this pull request:
- [x] Add exchange rate for all countries

@sagararyal @norbertschuler 
min

### CODE REVIEW

**Comment by captainamiedi** [comment]
To test the branch, you can set country to Argentina. Will finish up the country exchange rate today

**Comment by norbertschuler** [comment]
Please change this PR to draft (there is a link) so it does not get merged until it's ready to do so.

**Comment by sagararyal** [comment]
@captainamiedi   

Donate button is not loading when min Amt = entered amount.

Screen Shot 2020-12-29 at 1 53 58 PM

**Comment by captainamiedi** [comment]
> @captainamiedi
> 
> Donate button is not loading when min Amt = entered amount.
> 
> Screen Shot 2020-12-29 at 1 53 58 PM

I have fix these issue. You can test again. Thanks

File: src/utils/countryCurrency/countryExchange.json [hunk 1]
```diff
@@ -0,0 +1,673 @@
...
+        "value": 2.515023
+    },
+    {
+        "currencyCode": "FKP",
+        "value": 0.909639
+    },
+    {
+        "currencyCode": "GBP",
+        "value": 0.909479
+    },
```
Comment (norbertschuler) [inline src/utils/countryCurrency/countryExchange.json:199]: Currency exchange rates always change. How do we make sure we adapt these static file in the frontend over time (after a few month or years)?

File: src/utils/countryCurrency/getExchange.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import countryExchange from './countryExchange.json';
+
+export const getExchangeValue = (curreny) => {
```
Comment (norbertschuler) [inline src/utils/countryCurrency/getExchange.js:3]: I understood we use these values only for calculating the minimum amount to donate, but we 
- should make sure nobody else is using these values in the frontend for the purpose of showing real donation amounts in other currencies!
- make sure exchange rates get adapted from time to time

So maybe we can add some comment in this file to describe the purpose and constraints? Or we rewrite this function to `getMinimumAmountForCurrency()` containing the calculation `Math.round(exchange.value * 2)`, so nobody ever gets the idea of using `getExchangeValue()` for any other purpose.

**Review by norbertschuler — DISMISSED** [review]

**Comment by norbertschuler** [comment]
- [ ] Is it wanted that the donation button keep being shown with the loading animation if you are below the minimum donation amount? (No error, just a design question.)

Bildschirmfoto 2020-12-30 um 12 20 38

File: src/utils/countryCurrency/countryExchange.json (hunk 1 above)
Comment (captainamiedi) [reply src/utils/countryCurrency/countryExchange.json:199]: I had the same concern but @sagararyal said the fluctuations are not much. we can round off them  nearest number

File: src/utils/countryCurrency/getExchange.js (hunk 2 above)
Comment (sagararyal) [reply src/utils/countryCurrency/getExchange.js:3]: I agree with Norbert. When I shared the sheet with @captainamiedi  I asked him to already multiply the value in the json file.

Comment (captainamiedi) [reply src/utils/countryCurrency/getExchange.js]: Thanks @norbertschuler for these feedback. i have made changes to these effect. Kindly review and give feedback.

File: src/utils/countryCurrency/countryExchange.json (hunk 1 above)
Comment (norbertschuler) [reply src/utils/countryCurrency/countryExchange.json:199]: To name an example Thai Baht had a fluctuation between 34 up to 37 Baht for 1 € last year. 

B5DC795B-5996-452E-B672-8C4EFD1CAC6C

Comment (sagararyal) [reply src/utils/countryCurrency/countryExchange.json:199]: @norbertschuler this should be fine, as we are requiring min amount as 2€ equivalent. stripe min is around 1 € or less so these fluctuations do not affect transactions.

**Comment by sagararyal** [comment]
@harshvitra could you take over this PR, and also design fixes.

todo: While using min amount keep the amount multiplied by 2 in the json file. and fix minor design issues.

File: src/features/donations/screens/TreeDonation.tsx [hunk 3]
```diff
@@ -135,242 +139,254 @@ function TreeDonation({
...
+                  <ButtonLoader />
+                </div>
+              ) : (
+                <div className={styles.finalTreeCount}
+                  style={{ marginTop: '5px' }}
+                >
+                  <div className={styles.totalCostText}
+                    style={{ fontWeight: 'unset' }}
+                  >
+                    <p>{t('donate:minDonate')}</p>
```
Comment (norbertschuler) [inline src/features/donations/screens/TreeDonation.tsx:363]: space missing between text and amount

Comment (norbertschuler) [reply src/features/donations/screens/TreeDonation.tsx:363]: fixed it

**Review by norbertschuler — APPROVED** [review]
LGFM

**Pull request merged**